src/Unleash.php

```diff
@@ -14,4 +14,8 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
     public function getVariant(string $featureName, ?Context $context = null, ?Variant $fallbackVariant = null): Variant;
 
     public function register(): bool;
+
+    public function getFeatures(): array;
+
+    public function getFeature(string $featureName);
```

Comment:
Given this is a proxy method to the `findFeature()` method in repository, it should be named the same.